rest/context.go

```diff
@@ -130,12 +133,13 @@ type RequestContext interface {
 type gorillaRequestContext struct {
 	context.Context
 	req      *http.Request
+	router   *mux.Router
 	messages []string
 }
 
 // NewContext returns a RequestContext populated with parameters from the request path and
 // query string.
-func NewContext(parent context.Context, req *http.Request) RequestContext {
+func NewContext(parent context.Context, req *http.Request, router *mux.Router) RequestContext {
```

Comment:
This signature change breaks a lot of existing code. I haven't attempted a fix for any repos, so it may be a trivial issue. Just something to recognize. Some devs outside of Workiva will probably see broken builds as well.